Daf.Meta.sln

```diff
@@ -20,6 +20,32 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Daf.Meta.Editor.Tests", "Daf.Meta.Editor.Tests\Daf.Meta.Editor.Tests.csproj", "{23B76CA7-1702-42A4-8A7D-AEBCBDD3C1B9}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Sample Metadata", "Sample Metadata", "{46AB51C3-C177-4F29-97CA-AAFD7590B644}"
```

Comment:
This file shouldn't have any edits for this, the sample project should not be included in the solution.